client/src/styles/global.css

```diff
@@ -0,0 +1,53 @@
+
+/* global.css */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --cyf-color: #ff1e1e;
+  --white: #ffffff;
+  --gray: #808080;
+  --black: #000000:
+}
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--black);
+  background-color: var(--white);
+}
+
+a {
+  color: blue;
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 24px;
```

Comment:
Setting font size as pixels breaks accessibility we talked about copying CYF font sizes